rusoto/credential/src/profile.rs

```diff
@@ -20,6 +20,18 @@ pub struct ProfileProvider {
 }
 
 impl ProfileProvider {
+
+    /// Get the default profile name:
+    /// 1. if set use value from environment variable ```AWS_PROFILE```
+    /// 2. otherwise return ```"default"```
+    /// see https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html.
+    fn default_profile_name() -> String {
```

Comment:
Just a nitpick, but I'm wondering whether we should move this down to sit next to `default_profile_location`.

Reply:
OK, I grouped the constructor and helper functions accordingly.